web-registry/src/lib/ledger.ts

```diff
@@ -4,24 +4,10 @@ export const chainId = process.env.REACT_APP_LEDGER_CHAIN_ID;
 export const nctBasket = process.env.REACT_APP_NCT_BASKET === 'true';
 export const isBridgeEnabled = process.env.REACT_APP_BRIDGE === 'true';
 
-// expLedger is a feature flag indicating the app is connected to a testnet with experimental features (Hambach).
-export const expLedger = process.env.REACT_APP_EXP_LEDGER;
-
-// v4Ledger is a feature flag indicating the app is connected to a testnet with Regen Ledger v4.0.
-// It can be removed once hambach/redwood/mainnet upgrades to v4.0.
-export const v4Ledger = process.env.REACT_APP_V4_LEDGER;
-
-const expUriBase = 'exp-';
-const v4UriBase = 'v4-';
-const uriBase = `${getApiUri()}/${
-  v4Ledger ? v4UriBase : expLedger ? expUriBase : ''
-}`;
+const uriBase = `${getApiUri()}/}`;
```

Comment:
i get 404 when going to https://deploy-preview-1792--regen-registry.netlify.app/, seems due to the ledger proxy url which is set to https://regen-registry-server-staging.herokuapp.com/%7Dledger

```suggestion
const uriBase = `${getApiUri()}/`;
```

Reply:
Nice catch, it's fixed.